langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibTableTest.java

```diff
@@ -123,8 +123,18 @@ public void testHashCollisionHandlingScenarios() {
     }
 
     @Test
-    public void testHashCollisionInQuery() {
-        BRunUtil.invoke(compileResult, "testHashCollisionInQuery");
+    public void testHashCollisionInQueryWithAdd() {
```

Comment:
Shall we use a `DataProvider` for the tests? As we have the same pattern in all these tests.